src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1136,13 +1136,16 @@ namespace winrt::TerminalApp::implementation
         {
             _lastTabClosedHandlers(*this, nullptr);
         }
-        else if (_isFullscreen || _rearranging)
+        else if (_isFullscreen || _rearranging || _isInFocusMode)
```

Comment:
idle thought: _should this just be checking the visibility of TabView versus `Collapsed`? lol_